### ISSUE TITLE
Context menu: aria-label inconsistent in demo

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#839]
### Expected behavior

aria-label="Context menu" consistently across framework. 
### Actual behavior

The first context menu example has aria-label="Some label". The others are correct and would be good to be consistent so this doesn't end up in products.

**Comment by Blackbaud-PatrickOFriel** [comment]
The reason for that is to show that you can set a custom aria label for the context menu. What would be a better example of this? "Custom context menu"?

**Comment by blackbaud-johnly** [comment]
Would something like "Context menu label for screen readers" work?

**Comment by Blackbaud-MattGregg** [comment]
No. I think we should label this menu something consistent across the framework so when someone encounters it, they can hopefully leverage their past experience with it to understand what it is. It is labeled as "context menu" in the other two examples in this context menu component as well as all of the other components where I'd seen the context menu used - cards, grid, repeater.

**Comment by Blackbaud-PatrickOFriel** [comment]
I disagree. "Context menu" is the default label for this control. We allow them to have an escape hatch to have a custom label aria-label for this control if more label information would be needed. It is an optional label they can add if they wish (just like any aria label in any web page in the universe).

**Comment by Blackbaud-MattGregg** [comment]
I understand it can be changed if there is some use or circumstance which would warrant something more specific, but this menu is generally used the same way for the same thing. It does have to have a text label to communicate what it is or does. Otherwise, it would just read as "button". I'll stop by as it might be easier to discuss.

**Comment by Blackbaud-PatrickOFriel** [comment]
After discussing this with @Blackbaud-MattGregg, I explained that the reason that one of the `aria-labels` is different is to display the `bb-context-menu-label` functionality of the context menu directive. I will close this issue when we've updated the documentation for `bb-context-menu-label` to indicate that providing this label is optional, and will be set to "Context menu" by default.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.6](https://github.com/blackbaud/skyux/releases/tag/v1.7.6)
